packages/tasks/src/library-ui-elements.ts

```diff
@@ -774,6 +782,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: Partial<Record<ModelLibraryKey, Librar
 		docsUrl: "https://huggingface.co/docs/hub/ml-agents",
 		snippets: mlAgents,
 	},
+	"unity-sentis": {
+		btnLabel: "unity sentis",
+		repoName: "unity-sentis",
+		repoUrl: "https://github.com/huggingface/unity-sentis",
```

Comment:
This doesn't exist (the `ml-agents` example above is a fork, and it's going to be replaced in #439. Maybe you'd wnat to use https://github.com/Unity-Technologies/sentis-samples instead?

Reply:
Yes it can be interesting to add a Sentis documentation, as we did with ML-Agents: https://huggingface.co/docs/hub/ml-agents

Reply:
Hi, yes I can change the github link to the samples link. (I wasn't sure if that github gets created automatically... apparently not) .

It would be good to have Sentis documentation (how do we do this?). We have our own documentation too: https://docs.unity3d.com/Packages/com.unity.sentis@latest

Reply:
If you can provide the text that's awesome 🔥 We can add a page similar to the one @simoninithomas mentioned and link from there to your existing docs.

Reply:
I put together[ some text for the doc page](https://docs.google.com/document/d/1Er5PFY5oy9j_OWeQKjyhLrZZyiTWPM1HsRsQUwR_BdM/edit?usp=sharing)

Is this OK? Do we add it ourselves or do you add it? Thanks.

Reply:
You can open a PR directly to `hub-docs` such as this file https://github.com/huggingface/hub-docs/blob/main/docs/hub/asteroid.md